app/cdap/components/Datasets/iconStore.js

```diff
@@ -0,0 +1,88 @@
+      fill="#4681F4"
+    />
+  </svg>
+);
+
+export const CanBrowseIcon = () => (
+  <svg width="8" height="16" viewBox="0 0 8 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path opacity="0.8" d="M1 1L7 8L1 15.5" stroke="#5F6368" strokeLinecap="round" />
+  </svg>
+);
+
+export const CanBrowseIconHover = () => (
+  <svg width="8" height="16" viewBox="0 0 8 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path opacity="0.8" d="M1 0.5L7 7.5L1 15" stroke="white" strokeLinecap="round" />
+  </svg>
+);
+
+export const DownloadIcon = () => {
+  return (
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M20 12L18.59 10.59L13 16.17V4H11V16.17L5.42 10.58L4 12L12 20L20 12Z"
+        fill="black"
+        fillOpacity="0.54"
+      />
+      <path d="M1.5 17.5V23H22V17.5" stroke="#757575" strokeWidth="1.5" />
+    </svg>
+  );
+};
+
+export const PrevPageIcon = () => {
+  return (
+    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M16 7H3.83L9.42 1.41L8 0L0 8L8 16L9.41 14.59L3.83 9H16V7Z"
+        fill="black"
+        fillOpacity="0.54"
+      />
+    </svg>
+  );
+};
+
+export const SearchIcon = () => (
+  <svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M12.755 11.2021H11.965L11.685 10.9321C12.665 9.79215 13.255 8.31215 13.255 6.70215C13.255 3.11215 10.345 0.202148 6.755 0.202148C3.165 0.202148 0.255005 3.11215 0.255005 6.70215C0.255005 10.2921 3.165 13.2021 6.755 13.2021C8.365 13.2021 9.845 12.6121 10.985 11.6321L11.255 11.9121V12.7021L16.255 17.6921L17.745 16.2021L12.755 11.2021ZM6.755 11.2021C4.26501 11.2021 2.255 9.19215 2.255 6.70215C2.255 4.21215 4.26501 2.20215 6.755 2.20215C9.245 2.20215 11.255 4.21215 11.255 6.70215C11.255 9.19215 9.245 11.2021 6.755 11.2021Z"
+      fill="black"
+      fillOpacity="0.54"
+    />
+  </svg>
+);
+
+export const GCSIcon = () => {
```

Comment:
What is the plan for supporting icons for plugins that are added later?

Reply:
this will be covered as part of next milestone deliverable.